index.js

```diff
@@ -14,14 +20,14 @@ module.exports = {
       debugMode = process.env.EMBER_ENV === 'development';
     }
 
-    if (!process.env.EMBER_CLI_FASTBOOT) {
-      if (debugMode) {
-        app.import('bower_components/plupload/js/moxie.js');
-        app.import('bower_components/plupload/js/plupload.dev.js');
-      } else {
-        app.import('bower_components/plupload/js/plupload.full.min.js');
-      }
+
+    if (debugMode) {
+      app.import('vendor/plupload/moxie.js');
```

Comment:
This is the path to your file. Since we needed to wrap the file before importing it, we did it in treeForVendor and merged it there so it will be in the vendor trees.